packages/salesforcedx-vscode-core/src/commands/forceSourceDeploySourcePath.ts

```diff
@@ -135,13 +134,17 @@ export class LibraryDeploySourcePathExecutor extends LibraryCommandletExecutor<
         'force_source_deploy_with_sourcepath_beta'
       );
 
-      if (this.orgConnection === undefined) {
-        throw new Error('Connection is not established');
+      if (this.sourceClient === undefined) {
+        throw new Error('SourceClient is not established');
```

Comment:
Should we telemetry this?

Reply:
yeah, this is already taken care of by the catch in line 149